src/arrayMethodSort.js

```diff
@@ -1,11 +1,41 @@
 'use strict';
 
+const compareAsStrings = (a, b) => {
+  const stringA = String(a);
+  const stringB = String(b);
+
+  if (stringA > stringB) {
+    return 1;
+  } else if (stringA === stringB) {
```

Comment:
else is redundant here (after return statement in the previous one)
just create separate if statement without else